styles.css

```diff
@@ -0,0 +1,555 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+figure,
+blockquote,
+dl,
+dd {
+  margin: 0;
+}
+
+ul[role='list'],
+ol[role='list'] {
+  list-style: none;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+body {
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  background-color: rgb(248, 248, 248);
+  font-size: 16px;
+}
+
+img,
+picture {
+  max-width: 100%;
+  display: block;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+a:not([class]) {
+  text-decoration-skip-ink: auto;
+}
+
+button {
+  cursor: pointer;
+}
+
+.aboutTitle {
+  padding: 0 16px;
+}
+
+.displayNone {
+  display: none;
+}
+
+.displayNoneSmall {
+  display: none;
+}
+
+.menuList {
+  display: none;
+}
+
+.playerCard {
+  display: none;
+}
+
+.listNone {
+  list-style-type: none;
+}
+
+.flex {
+  display: flex;
+}
+
+.justifyCenter {
+  justify-content: center;
+}
+
+.flexWrap {
+  flex-wrap: wrap;
+}
+
+.justifySpaceBetween {
+  justify-content: space-between;
+}
+
+.flexAlignCenter {
+  align-items: center;
+}
+
+.flex110 {
+  flex: 1 1 0;
+}
+
+.flex330 {
+  flex: 3 3 0;
+}
+
+.primary {
+  color: #ec5242;
+}
+
+.secondary {
+  color: #272a31;
+}
+
+.tertiary {
+  color: #d3d3d3;
+}
+
+.white {
+  color: white;
+}
+
+h3 {
+  font-size: 24px;
+  font-weight: 400;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 900;
+}
+
+h2 {
+  font-size: 30px;
+  font-weight: 900;
+}
+
+.italic {
+  font-style: italic;
+}
+
+.fontSize12 {
+  font-size: 12px;
+}
+
+.fontsize14 {
+  font-size: 14px;
+}
+
+.fontSize18 {
+  font-size: 18px;
+}
+
+.fontSize20 {
+  font-size: 20px;
+}
+
+.fontSize24 {
+  font-size: 24px;
+}
+
+.fontSize36 {
+  font-size: 36px;
+}
+
+.fontWeight700 {
+  font-weight: 700;
+}
+
+.mainProgram {
+  background-color: #272a31;
+}
+
+.lineHeight15 {
+  line-height: 1.5;
+}
+
+.mLeft20 {
+  margin-left: 20px;
+}
+
+.mRight20 {
+  margin-right: 20px;
+}
+
+.hamburger {
+  padding-left: 32px;
+  padding-top: 24px;
+  padding-bottom: 24px;
+}
+
+.welcomeText {
+  margin-top: 40px;
+}
+
+.subText {
+  margin: 20px 10px;
+  padding: 10px;
+  border: 3px solid white;
+}
+
+.location {
+  margin-top: 10px;
+  margin-bottom: 100px;
+}
+
+.program {
+  padding-top: 36px;
+  padding-bottom: 48px;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly make sure all the necessary sections of your webpage has the redline aligned in the center of your webpage 🙏 

Original Behance Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190200163-97bb13f3-4264-4ebf-8ae7-f081d7f2267f.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190200308-85e1f97d-0b37-4e43-9e1c-0a57610f2c64.png)